Tests/t_ldapobject.py

```diff
@@ -371,6 +371,62 @@ def test_multiple_starttls(self):
             l.simple_bind_s(self.server.root_dn, self.server.root_pw)
             self.assertEqual(l.whoami_s(), 'dn:' + self.server.root_dn)
 
+    def assert_option_equal(self, conn, option, value):
+        self.assertEqual(conn.get_option(option), value)
+
+    @requires_tls()
+    def test_set_tls_options_ldap(self):
```

Comment:
Looks like the coverage report has failed.
I think it's important to test all the parts before merging... 

If you haven't started on the test suite expansion yet, I can take a look later and add a few tests so it will cover the rest of the code. :)